utils/index.js

```diff
@@ -29,11 +29,10 @@ exports.getReactScriptsVersion = function getReactScriptsVersion(cliVersion) {
   }
 
   const { dependencies } = packageJson;
-  const reactScriptsVersionString = /^[~^]?([.0-9]+)$/.exec(dependencies['react-scripts'])[1];
   const versions = {
-    major: Number(semver.major(reactScriptsVersionString)),
-    minor: Number(semver.minor(reactScriptsVersionString)),
-    patch: Number(semver.patch(reactScriptsVersionString)),
+    major: Number(semver.major(dependencies['react-scripts'])),
+    minor: Number(semver.minor(dependencies['react-scripts'])),
+    patch: Number(semver.patch(dependencies['react-scripts']))
```

Comment:
Your IDE seems to be removing the trailing comma, breaking the code styles and adding noise to the commit.

Reply:
Don't worry about it, as I'll work on the file again I'll change them myself.